### ISSUE TITLE
深度学习-pytorch-经典CNN模型-NIN和GoogLeNet | 蜡笔小金QAQ的个人主页

### ISSUE THREAD

**Issue by MrKiing-Jin** [MrKiing-Jin/MrKiing-Jin.github.io#18]
https://www.mrking.space/2022/08/22/CNN_NIN/ 

 NIN网络结构 代码实现1234567891011import torchfrom torch import nnfrom d2l import torch as d2ldef nin_block(in_channels, out_channels, kernel_size, strides, padding):    return nn.Sequential(        nn.Conv2d(
